### PR TITLE
Fix/adis16448 bmlz fixes

### PR DESCRIPTION
**Pull request by ASM3**
This PR solves some (timing) compatibility issues which appeared on the ADIS16448BMLZ version.

### CODE REVIEW

File: src/drivers/adis16448/adis16448.cpp [hunk 1]
```diff
@@ -293,6 +293,21 @@ class ADIS16448 : public device::SPI
 		uint16_t		mag_z;
 		uint16_t		baro;
 		uint16_t		temp;
+
+		ADISReport():
+			cmd(0),
```
Comment (dagar) [inline src/drivers/adis16448/adis16448.cpp]: Minor suggestion, initialize these right in the class definition. It's much easier to do a pass and ensure you've got them all, and you don't have to deal with ordering.

File: src/drivers/adis16448/adis16448.cpp [hunk 2]
```diff
@@ -782,9 +798,9 @@ ADIS16448::_set_sample_rate(uint16_t desired_sample_rate_hz)
 		smpl_prd = BITS_SMPL_PRD_NO_TAP_CFG;
 	}
 
-	modify_reg16(ADIS16448_SMPL_PRD, 0x0700, smpl_prd);
+	modify_reg16(ADIS16448_SMPL_PRD, 0x1f00, smpl_prd);
```
Comment (dagar) [inline src/drivers/adis16448/adis16448.cpp]: What's the magic value? Should it be a DEFINE?

**Comment by dagar** [comment]
Minor suggestions, but it's still good to merge as is. Again, I don't have the hardware to actually verify this works.

**Comment by yangang123** [comment]
@ASM3 

I have an ununderstood question about the adis1648 driver, url: https://github.com/PX4/Firmware/issues/8525

thanks!!

**Pull request merged**